Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/profil/ProfileActivity.java

```diff
@@ -1,15 +1,59 @@
 package com.bounswe2017.group10.atlas.profil;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
 
 import com.bounswe2017.group10.atlas.R;
+import com.bounswe2017.group10.atlas.auth.AuthActivity;
+import com.bounswe2017.group10.atlas.httpbody.UserResponse;
+import com.bounswe2017.group10.atlas.remote.APIUtils;
+import com.bounswe2017.group10.atlas.util.Constants;
+import com.bounswe2017.group10.atlas.util.Utils;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+import static com.bounswe2017.group10.atlas.util.Utils.getSharedPref;
 
 public class ProfileActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_profile);
+
+        String authStr = getSharedPref(this).getString(Constants.AUTH_STR, Constants.NO_AUTH_STR);
+        APIUtils.serverAPI().getMe(authStr).enqueue(new Callback<UserResponse>() {
+            @Override
+            public void onResponse(Call<UserResponse> call, Response<UserResponse> response) {
+                if (response.isSuccessful()) {
+                    UserResponse body = response.body();
+                    ((TextView) findViewById(R.id.user_profile_name)).setText(body.getFirstname()+" "+body.getLastname());
+                    ((TextView) findViewById(R.id.user_profile_email)).setText(body.getEmail());
+                } else {
+                    ((TextView) findViewById(R.id.user_profile_name)).setText("name");
+                    ((TextView) findViewById(R.id.user_profile_email)).setText("email");
+                }
```

Comment:
1. You can get TextView getting out of the if-else statements.
2. We need to show a Toast if the response is not successful.